### PR TITLE
exui-1717-show warning if task not available

### PR DESCRIPTION
**Pull request by RiteshHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EXUI-1717

### Change description ###
show warning it task not available

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by olusegz07 — DISMISSED** [review]

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/work-allocation.service.ts [hunk 1]
```diff
@@ -96,6 +96,8 @@ export class WorkAllocationService {
    */
   public completeTask(taskId: string): Observable<any> {
     if (!this.isWAEnabled()) {
+      this.alertService.setPreserveAlerts(true);
+      this.alertService.warning({ phrase:'WA Enabled: A task could not be completed successfully. Please complete the task associated with the case manually.'});
```
Comment (andywilkinshmcts) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/work-allocation.service.ts:100]: I would change this to "completeTask: Work Allocation is not enabled, so the task could not be completed. Please complete the task associated with the case manually."

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/work-allocation.service.ts [hunk 2]
```diff
@@ -119,6 +121,8 @@ export class WorkAllocationService {
    */
   public assignAndCompleteTask(taskId: string): Observable<any> {
     if (!this.isWAEnabled()) {
+      this.alertService.setPreserveAlerts(true);
+      this.alertService.warning({ phrase:'WA Enabled: A task could not be completed successfully. Please complete the task associated with the case manually.'});
```
Comment (andywilkinshmcts) [inline projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/work-allocation.service.ts:125]: I would change this to "assignAndCompleteTask: Work Allocation is not enabled, so the task could not be completed. Please complete the task associated with the case manually."

**Review by andywilkinshmcts — CHANGES_REQUESTED** [review]

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/work-allocation.service.ts (hunk 2 above)
Comment (RiteshHMCTS) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/work-allocation.service.ts]: updated

File: projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/work-allocation.service.ts (hunk 1 above)
Comment (RiteshHMCTS) [reply projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/services/work-allocation.service.ts]: updated

**Pull request closed without merging**